assets/js/signup.js

```diff
@@ -90,13 +113,18 @@ var questionRules = function(question) {
     rule["required"] = true;
   }
   if (question.validationType != null) {
+    // There should really be some form of validation for the validationType
+    // One option would be to make a new table with all of the possible validation types
+    // which would be both a foreign key for the validationType field in the questionInformation table
+    // and a way to validate at this point too. If the validationType has a typo for any
+    // given field, the front-end form validation fails to run at all for any of the fields.
```

Comment:
The validation methods include...

required, remote, email, url, date, dateISO, number, digits, equalTo, maxlength, minlength, rangelength, range, max, min, step
as found in https://github.com/jquery-validation/jquery-validation/blob/master/src/core.js

as well as 50 others found in https://github.com/jquery-validation/jquery-validation/tree/master/src/additional

That's 66 different possible validation types.

We would just need to check if the validationType we get back from the database is one of those strings, and then add it to the rule array if it is or don't if it isn't.